include/chemfiles/Format.hpp

```diff
@@ -37,23 +43,25 @@ class CHFL_EXPORT Format {
     /// @param frame The frame to fill
     virtual void read_step(size_t step, Frame& frame);
 
-    /// @brief Read a specific step from the trajectory file.
+    /// Read the next step from the trajectory file.
```

Comment:
Should it be `step` instead of step?

Reply:
with backticks? I don't think so, see the answer above.

Reply:
Makes sense now.